src/test/java/it/pagopa/transactions/services/TransactionServiceTest.java

```diff
@@ -99,10 +101,10 @@ void shouldHandleNewTransactionTransactionActivated() throws Exception {
          */
         Mockito.when(transactionActivateHandler.handle(Mockito.any(TransactionActivateCommand.class)))
                 .thenReturn(Mono.just(response));
-//        Mockito.when(transactionsProjectionHandler.handle(transactionActivationRequestedEvent)).thenReturn(Mono.just(transactionActivationRequested));
         Mockito.when(transactionsActivationProjectionHandler.handle(transactionActivatedEvent))
                 .thenReturn(Mono.just(transactionActivated));
-
+        Mockito.when(transactionsUtils.convertEnumeration(any()))
+                .thenAnswer(args -> TransactionStatusDto.fromValue(args.getArgument(0).toString()));
```

Comment:
In this case probably `callRealMethod` would be more appropriate

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/169/commits/d06f51375cab419c5e9049e85e71b60c0f582244